openedx/features/course_experience/__init__.py

```diff
@@ -73,16 +72,16 @@
 )
 
 # .. toggle_name: course_experience.relative_dates
-# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_implementation: CourseWaffleFlag
 # .. toggle_default: False
 # .. toggle_description: Waffle flag to enable relative dates for course content. A 'Dates' tab will be visible in the
-#   course view showing key course dates.
+#   course view showing key course dates. Was previously an ExperimentWaffleFlag with experiment_id=17.
 # .. toggle_use_cases: opt_in
 # .. toggle_creation_date: 2020-02-10
 # .. toggle_warnings: To set a relative due date for self-paced courses, the weeks_to_complete field for a course run
 #   needs to be set. Currently it can be set through the publisher app.
 # .. toggle_tickets: https://openedx.atlassian.net/browse/AA-27
-RELATIVE_DATES_FLAG = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'relative_dates', __name__, experiment_id=17)
+RELATIVE_DATES_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'relative_dates', __name__)
```

Comment:
Huh, I thought we had discussed changing this to an opt-out flag (default True and/or change the name). But I don't see that represented in the ticket. So maybe I just made that up.

I think that would be a good change so that open edx gets this benefit out of the box, but it was not in the ticket, so :shrug:

Reply:
I think that might be a part of a separate effort? This change was purely to stop the bucketing events since they dominate our experiment data in segment and at this point, all users are bucketed into the experiment.